src/app/view-questions/view-questions.component.html

```diff
@@ -0,0 +1,28 @@
+<app-header></app-header>
+
+<div class="container">
+  <h2>Asked Questions</h2>
+  <div class="list-group">
+    <a href="#" class="list-group-item">
```

Comment:
what are this for? are they the answers for the asked question? if so make them a component and loop over them as we did for question